nntrainer/layers/bn_layer.cpp

```diff
@@ -27,30 +27,30 @@
 #include <nntrainer_error.h>
 #include <nntrainer_log.h>
 #include <node_exporter.h>
+#include <normalization.h>
 #include <util_func.h>
 
 namespace nntrainer {
 
 static constexpr size_t SINGLE_INOUT_IDX = 0;
 
 enum BNParams {
-  mu,
-  var,
+  moving_mean,
+  moving_variance,
   gamma,
   beta,
   deviation,
-  invstd,
-  cvar,
-  t_reduced,
-  t_full
+  variance,
```

Comment:
These order needs to be remain still to make consistency with other exporter.

Reply:
I will split the commit and make another pr as you recommended. This code will be included in there.